ipynb/scripts/remove-blooms.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+import qiime2 as q2
+from biom import load_table, Table
+
+bt = q2.Artifact.load('deblur-fmt-and-icu/table.qza').view(Table)
+
+blooms = []
+with open('newbloom.all.fna', 'r') as f:
+    for line in f.read().split('\n'):
+        if line.startswith('>'):
+            pass
+        else:
+            blooms.append(line.strip())
+blooms = [b[:125] for b in blooms]
```

Comment:
possible to add a comment noting that this is the trim length used for the manuscript as one sequencing round was at 125 cycles?